trade-finance-manager-ui/styles/table.scss

```diff
@@ -156,4 +156,26 @@ table.premium-payments-table {
   .payments-list-item {
     list-style-type: none;
   }
-}
\ No newline at end of file
+}
+
+table.utilisation-report-reconciliation-table {
+  table-layout: fixed;
+
+  .ukef-width-one-tenth {
+    width: 10% !important;
```

Comment:
These live inside the table (vs in `layout.css`) as they shouldn't be used for general grid styling